### PR TITLE
refs #13528 - removing unused apipie doc'd param

### PR DESCRIPTION
**Pull request by komidore64**

### CODE REVIEW

File: app/controllers/katello/api/v2/repositories_controller.rb [hunk 1]
```diff
@@ -43,7 +43,7 @@ class Api::V2::RepositoriesController < Api::V2::ApiController
     param :erratum_id, String, :desc => N_("Id of an erratum to find repositories that contain the erratum")
     param :rpm_id, String, :desc => N_("Id of a package to find repositories that contain the rpm")
     param :library, :bool, :desc => N_("show repositories in Library and the default content view")
-    param :content_type, String, :desc => N_("limit to only repositories of this time")
+    param :content_type, String, :desc => N_("limit to only repositories of type 'yum', 'puppet', or 'docker'")
```
Comment (beav) [inline app/controllers/katello/api/v2/repositories_controller.rb]: also ostree

**Comment by beav** [comment]
APJ after 'ostree' is added

File: app/controllers/katello/api/v2/repositories_controller.rb (hunk 1 above)
Comment (daviddavis) [inline app/controllers/katello/api/v2/repositories_controller.rb]: Would it make sense to change this field to a set of options like https://git.io/vglM0?

Comment (komidore64) [inline app/controllers/katello/api/v2/repositories_controller.rb]: @daviddavis oh, yeah. that does make a lot more sense.

Comment (komidore64) [inline app/controllers/katello/api/v2/repositories_controller.rb]: @beav ostree stuff hasn't been merged into master yet.

Comment (beav) [inline app/controllers/katello/api/v2/repositories_controller.rb]: @komidore64 you are right. I like the idea of the option set but APJ from me.

Comment (komidore64) [inline app/controllers/katello/api/v2/repositories_controller.rb]: @beav, @daviddavis, well..., i guess the changes to this file are kinda outside the scope of the rest of the PR?

**Comment by komidore64** [comment]
@Katello/katello-core one more ack from someone?

**Comment by ehelms** [comment]
ACK

**Pull request merged**